### DISCUSSION TITLE
How to import a PSD/Procreate file ?

### DISCUSSION THREAD

**Discussion by generic-matrix** [keiwando/toonsquid#56] [General]
Hello team,

I am trying to import a PSD file from Procreate but it does not show up on the app .

Thank you

**Comment by keiwando** [comment]
You can now import and export PSD files with the ToonSquid 2.0.9 update:
https://toonsquid.com/updates/ToonSquid-2_0_9/

Old answer:
~Hi! PSD imports are currently not supported.~
~I've done some work on it a while ago but will have to go back at some point and finish it if enough people ask for it.~

**Reply by folklore15** [reply]
Omg! Pls do that! 
Smooth workflow between Callipeg and Photoshop is the last reason why Callipeg is more useful than ToonSquid! 
ToonSquid with support psd export/import... im wet already @q@
<3

**Reply by Yelisava** [reply]
Support the wetness.

**Reply by potatpengu** [reply]
Please work on psd import/export again. it would be a great time saver for me. i wouldn’t need to import every single cut section and reorganize them for key frame animations.

**Reply by Forikundo** [reply]
Psd import would be awesome! :))

**Reply by spelljammed** [reply]
Just adding my voice to this. PSD import is an essential feature for my workflow! I build my assets in Photoshop/Procreate, and being able to quickly import a many-layered PSD file (with layer groups/folders intact!) would be huge for me. I really hope this can be implemented.

**Reply by Cr8iveMike** [reply]
I'm raising my hand in support for PSD import support features. It will help speed up workflow between apps and efficiently. Thanks for your hard work so far.

**Comment by andydjames** [comment]
Export psd to preserve vectors.

**Comment by amoraleite** [comment]
Would be wonderful! But would be incredible if we can import vectors from another vector program like affinity design or illustrator. CRT c direct CRT v . 😃🙌🏻

**Comment by DoodlerPete** [comment]
Being able to import PSD files as transparent layers instead of individual PNG files would make ToonSquid fit into my workflow perfectly. I still love the app and will do my best to continue to use it, despite Procreate releasing DREAMS, in November. Keep up the awesome work.

**Comment by rowby** [comment]
One of the first things I checked into before getting ToonSquid was it supported importing of layered PSD Photoshop files.  Not happy that it did not — but I got ToonSquid anyway because I liked its other features and it was constantly being updated and improved.

Still would love to have PSD layer import.  And I’m sure many other potential customers would choose ToonSquid if that was a feature — word spread fast!

…Rowby.
